src/server/system_services/master_key_manager.js

```diff
@@ -43,19 +47,24 @@ class MasterKeysManager {
 
 
     get_root_key_id() {
-        return ROOT_KEY;
+        const { NOOBAA_ROOT_SECRET } = process.env;
+        return NOOBAA_ROOT_SECRET ? ROOT_KEY : undefined;
     }
 
     is_root_key(root_key_id) {
         return root_key_id && root_key_id.toString() === ROOT_KEY.toString();
     }
 
+    get_current_root_key_id() {
```

Comment:
I feel we should remove the old format or at least merge it so it won't be even more confusing

Reply:
this is still relevant